File: LeetCode/LeetCode Practice Problems/1374- Generate a String With Characters That Have Odd Counts.cpp
class Solution {
public:
    string generateTheString(int n) {
        string ans = "";
        for(int i=0;i<n-1;i++)
            ans = ans + 'a';
        
        if(n%2 == 0)
            ans = ans + 'b';
        else
            ans = ans + 'a';
        
        return ans;
    }
};

File: LeetCode/LeetCode Practice Problems/1374-Generate a String With Characters That Have Odd Counts.java
class Solution {
    public String generateTheString(int n) {
        String ans = "";
        int cnt = (n%2 == 1)?n:n-1;
        
        for(int i=1;i<=cnt;i++) ans = ans + "a";
        
        if(cnt < n) ans = ans + "b";
        return ans;
        
    }
}
